staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/typeconverter_test.go

```diff
@@ -68,7 +68,6 @@ metadata:
   labels:
     app: nginx
 spec:
-  replicas: 3
```

Comment:
Had to remove these from the test because it turns out the test was making sure we preserved the wrong behavior here